DelBaratico/src/main/java/cr/una/delbaratico/dao/AreaDAO.java

```diff
@@ -0,0 +1,46 @@
+package main.java.cr.una.delbaratico.dao;
+
+import main.java.cr.una.delbaratico.model.Area;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AreaDAO {
+
+    private JdbcUtil jdbcUtil;
```

Comment:
Each time a DAO is created, one connection is created to the database. So, every time you need to perform an operation in the data base you will need to spend resources creating an entirely new connection. Using this design wastes time and allows the scenario with several open database connections at the same time. I think it is better to create just one connection before the creation of any DAO and then pass it the connection (jdbcUtil object) through parameters. In that way, all DAOs can share the **same** connection, the one we created at the beginning. I think this is more efficient and quick design.